src/main/java/nearsoft/academy/bigdata/recommendation/MovieRecommender.java

```diff
@@ -30,57 +30,63 @@
     private int reviews = 0;
     private long totalProd = 0;
     private long totalUsers = 0;
-    HashMap<String, Long> productsTotal = new HashMap<String, Long>();
-    HashMap<Long, String> productsTotalReverse = new HashMap<Long, String>();
-    HashMap<String, Long> usersTotal = new HashMap<String, Long>();
-    ArrayList<String> products = new ArrayList<String>();
-    ArrayList<String> users = new ArrayList<String>();
-    ArrayList<String> scores = new ArrayList<String>();
+    HashMap<String, Long> productsTotal = new HashMap<>();
+    HashMap<Long, String> productsTotalReverse = new HashMap<>();
+    HashMap<String, Long> usersTotal = new HashMap<>();
```

Comment:
IMO, `usersTotal` name does not reflect the purpose of the variable. For me, it sounds that it will hold a unique numerical value instead of a Map